draft-ietf-quic-tls.md

```diff
@@ -753,9 +770,8 @@ thus ensuring that the keys are different for each version of QUIC. This
 prevents a middlebox that only recognizes one version of QUIC from seeing or
 modifying the contents of handshake packets from future versions.
 
-The HKDF function defined in TLS 1.3 MUST be used even in case the minimum TLS
-version that the endpoint is willing to use is greater, so as to assure that
-the peer can decrypt the packet.
+The HKDF-Expand-Label function defined in TLS 1.3 MUST be used for Initial
```

Comment:
Might be worth stating "For QUICv1 ..."

Reply:
Yeah, I considered that, but it's a long slide down that slope.  I think that it's sufficiently strongly implied.